locales/en.default.schema.json

```diff
@@ -1324,6 +1340,10 @@
         "show_vendor": {
           "label": "Show vendor"
         },
+        "show_rating": {
+          "label": "Show product rating",
+          "info": "To display a rating, add a product ratings app. [Learn more](https:\/\/help.shopify.com\/en\/manual\/products\/product-reviews\/installation)"
```

Comment:
Fact checking in the app store around "product ratings app" vs "product reviews app": 
- "product rating" returns 1465 app results
- "product reviews" returns 1523 results

Should we revise `product ratings app` to `product reviews app` in the text to capture more options?

"info": "To display a rating, add a product reviews app. [Learn more](https:\/\/help.shopify.com\/en\/manual\/products\/product-reviews\/installation)"